mne/time_frequency/tests/test_stockwell.py

```diff
@@ -38,8 +42,13 @@ def test_stockwell_check_input():
         assert_equal(zero_pad, 128 - last_dim)
 
 
-def test_stockwell_st_power_itc():
-    """Test stockwell power itc"""
+def test_stockwell_st_power_itc_power_of_2():
+    """
+    Test stockwell power itc
+    Handling of input data with 2**N samples per event was broken
+    Cf. https://github.com/mne-tools/mne-python/pull/3605
```

Comment:
Ok this is a lot, have a look at the other tests in other module. Typically they have one liner comments as these get printed on testing. A good one would be:

"""Test handling of input data with 2**N samples"""